src/main/scala/dpla/ingestion3/entries/ingest/IngestRemap.scala

```diff
@@ -0,0 +1,73 @@
+package dpla.ingestion3.entries.ingest
+
+import dpla.ingestion3.confs.{CmdArgs, Ingestion3Conf}
+import dpla.ingestion3.entries.ingest.EnrichEntry.executeEnrichment
+import dpla.ingestion3.entries.ingest.JsonlEntry.executeJsonL
+import dpla.ingestion3.entries.ingest.MappingEntry.executeMapping
+import dpla.ingestion3.entries.reports.ReporterMain.executeAllReports
+import dpla.ingestion3.utils.Utils
+import org.apache.spark.SparkConf
+
+/**
+  * Single entry point to run harvest, mapping, enrichment, indexing jobs and all reports.
+  * Support running against an already harvested data set and running a fresh harvest
+  *
+  * Expects four parameters:
+  *   1)  --output  A base path to save the mapping, enrichment, json-l and report outputs
+  *   2)  --input   A path to the harvested data
+  *   3)  --conf    A path to the application configuration file
+  *   4)  --name    Provider short name
+  */
+object IngestRemap {
```

Comment:
I find the name of this object a little confusing - `Ingest` or `IngestEntry` would be my instinct.  However, you're more familiar with the workflow than me, so I think you should choose whatever name you like.

Reply:
Yeah, I went back and forth on this. This entry point is just supposed to handle testing a mapping change (e.g. testing something on CQQ). I thought about CqaReingest but nothing really sounded great. 

Open to other names..

Reply:
`WholeShebang`